### ISSUE TITLE
Grass disappear after loading a save/Loading new Worldspace

### ISSUE THREAD

**Issue by Shazuli** [SaneEngineer/No-Grass-In-Objects-NG#4]
***Hi***

I generated the grass cache without issues, but I don't see any grass rendering any further than what it does in vanilla. I changed `Overwrite-grass-distance` (60000) and `Overwrite-grass-fade-range` (30000) to 10x normal but grass still fades away like it does in vanilla, `Use-grass-cache`, `Extend-grass-distance` and `Extend-grass-count` are all `true`. The only thing that changes with the mod installed is when you place a quicksave and then load it all the grass in the whole world disappears. Only a complete restart of the game returns grass like it used to be.

Skyrim AE: 1.6.1170
NGIO-NG: 1.0.8

**Comment by SaneEngineer** [comment]
Try the 1.0.9 and try setting `Overwrite-grass-distance` to `12000` and `Overwrite-grass-fade-range` to `6000`. Distances too large will cause issues.

**Comment by Shazuli** [comment]
```
[23:02:32:220] NGIO-NG v1.0.0.1
[23:02:32:220] Game version : 1-6-1170-0
[23:03:39:587] Created Cache for Raycasting Settings
[23:05:04:066] ChosenGrassSettings: grid: 3, dist: 6000, range: 3000, iMinGrassSize: 35
```

This is what it said last night. Updating to 1.0.9 it seems to be drawing grass further away and the new log also prints out the correct config change:

```
[11:27:36:633] NGIO-NG v1.0.0.1
[11:27:36:633] Game version : 1-6-1170-0
[11:28:43:523] Created Cache for Raycasting Settings
[11:29:54:349] ChosenGrassSettings: grid: 7, dist: 18000, range: 9000, iMinGrassSize: 35
```

It however still unloads all grass when loading a save (Quicksave, Autosave or New Save) or when loading a different interior (entering Whiterun or exiting Whiterun). A restart of the whole game is required to have it show up again.

**Comment by Shazuli** [comment]
Not sure if it'll help, but here's some of the mods I use that could possibly cause this.

Note that I re-cached the grass with 'seasonal grass' turned off to see if anything changed, which it did not so I doubt the seasons mods is a possible cause for this bug.

* [Skoglendi - A Grass Mod](https://www.nexusmods.com/skyrimspecialedition/mods/93944)

* [SkyrimSE Re-Engaged ENB](https://www.nexusmods.com/skyrimspecialedition/mods/1089)

* [Seasons of Skyrim SKSE](https://www.nexusmods.com/skyrimspecialedition/mods/62861)

* [Turn of the Seasons](https://www.nexusmods.com/skyrimspecialedition/mods/63623)

**Comment by SaneEngineer** [comment]
The issue only with the extend grass distance part of the mod. It is not the result of an conflict.
I'm testing a few fixes, I'll update here when I find a fix.

**Comment by SaneEngineer** [comment]
Should be fixed with 1.0.11.

**Comment by SaneEngineer** [comment]
All updates on the issue will be referenced in #30.
